android/app/build.gradle

```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:0.19.+"
+    compile "com.facebook.react:react-native:+"
```

Comment:
Why this change? I don't believe our plugin would support 0.18, so having a min version still seems to make sense.